### PR TITLE
mod_banners count integer to text

### PR DESCRIPTION
**Pull request by pe7er**
PR for https://github.com/joomla/joomla-cms/issues/5773

Changes in Extensions > Modules > Banner > "Count" option from
drop down box with values 1 to 10 (integer)
to free input field (text).

### CODE REVIEW

**Comment by dgt41** [comment]
It needs to validate only for integers (I think just adding `class="validate-number" will do the trick)

**Comment by dgt41** [comment]
`class="validate-numeric"` is the right one!

**Comment by dgt41** [comment]
@pe7er you have a PR for that!

**Comment by pe7er** [comment]
Great catch @dgt41! 
Thanks!

File: modules/mod_banners/mod_banners.xml [hunk 1]
```diff
@@ -40,10 +40,8 @@
 
 				<field
 					name="count"
-					type="integer"
-					first="1"
-					last="10"
-					step="1"
+					type="text"
+                                        class="validate-numeric"
```
Comment (zero-24) [inline modules/mod_banners/mod_banners.xml]: @pe7er your IDE change again tabs to spaces and add one space to much :smile:

Comment (pe7er) [inline modules/mod_banners/mod_banners.xml]: pff... I have to dig into my IDE to fix that annoying tab/space problem... 
Thanks for correcting me!

**Comment by Kubik-Rubik** [comment]
@pe7er Please also add a server side check with **filter="integer"**. Thank you!

**Comment by pe7er** [comment]
Ok, done. Thanks @Kubik-Rubik

**Comment by Kubik-Rubik** [comment]
Works as described! One more tester please.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5804">issues.joomla.org/joomla-cms/5804</a>.</sub>

**Comment by dgt41** [comment]
@test OK!

**Comment by Kubik-Rubik** [comment]
Thank you for testing it @dgt41!

**Pull request merged**